mmdeploy/codebase/mmdet/deploy/object_detection.py

```diff
@@ -316,6 +316,7 @@ def get_postprocess(self, *args, **kwargs) -> Dict:
                 params['mask_thr_binary'] = params['rcnn']['mask_thr_binary']
         if 'mask_thr_binary' in params:
             type = 'ResizeInstanceMask'  # for instance-seg
+            params['is_resize_mask'] = True  # resize and crop mask default
```

Comment:
```suggestion
            params['is_resize_mask'] = False  # resize and crop mask default
```

same reason as above

Reply:
👌